### PR TITLE
Extract cleanAST to language-specific files

### PR DESCRIPTION
**Pull request by azz**

### CODE REVIEW

**Comment by vjeux** [comment]
Nice! Can't wait to see it land :)

File: scripts/build/parsers.js [hunk 1]
```diff
@@ -1,12 +1,8 @@
...
-  "language-css/parser-postcss",
-  "language-html/parser-parse5",
-  "language-markdown/parser-markdown",
-  "language-vue/parser-vue"
-];
+const globby = require("globby");
+const path = require("path");
+
+module.exports = globby.sync("language-*/parser-*", {
+  cwd: path.join(__dirname, "../../src")
```
Comment (j-f1) [inline scripts/build/parsers.js]: This should be `path.join(__dirname, "..", "..", "src")`.

Comment (azz) [reply scripts/build/parsers.js]: Why's that?

**Comment by azz** [comment]
Moved to https://github.com/prettier/prettier/pull/3610

File: scripts/build/parsers.js (hunk 1 above)
Comment (j-f1) [reply scripts/build/parsers.js]: On Windows, the path separators will be `\`, so `../../src` won’t work.

Comment (azz) [reply scripts/build/parsers.js]: It gets normalised correctly. 

```
$ node -p 'path.win32.join(__dirname, "../code/prettier")'
..\code\prettier
```

**Pull request closed without merging**